thrift/src/main/thrift/contentatom.thrift

```diff
@@ -54,6 +54,10 @@ struct ContentChangeDetails {
 
 struct Flags {
   1: optional bool suppressFurniture
+  2: optional bool sensitive
+  3: optional bool legallySensitive
+  4: optional bool suppressRelatedContent
+  5: optional bool blockAds
```

Comment:
In the past we have discussed on deprecating `blockAds` as a rule should come from each renderer depending on the properties of the content rather than being decided by the content producer